Store.Web/Controllers/HomeController.cs

```diff
@@ -1,26 +1,29 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using Store.Models;
 using System.Diagnostics;
+using RockLib.Logging;
+using ILogger = RockLib.Logging.ILogger;
 
 namespace Store.Controllers
 {
-
     public class HomeController : Controller
     {
-        private readonly ILogger<HomeController> _logger;
+        private readonly ILogger _logger;
 
-        public HomeController(ILogger<HomeController> logger)
+        public HomeController(ILogger logger)
         {
             _logger = logger;
         }
 
         public IActionResult Index()
         {
+            _logger.Info("Starting home page");
             return View();
         }
 
         public IActionResult Privacy()
         {
+            _logger.Info("Opened Privacy page");
```

Comment:
If your website had 1,000,000 visitors a day, think about the amount of data this would generate. Is it worth it?